### PR TITLE
Add bilingual settings and Chinese translation

### PR DESCRIPTION
**Pull request by vickyhella**
Adds i18n settings and Chinese docs

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -39,6 +51,10 @@ module.exports = {
           label: 'Docs',
           className: 'navbar__docs',
         },
+        {
+            type: "localeDropdown",
+            position: "right",
+          },
```
Comment (nunix) [inline docusaurus.config.js:57]:
```suggestion
        {
          type: "localeDropdown",
          position: "right",
        },
```

**Review by nunix — COMMENTED** [review]
Fixed small typo, and the rest seems OK for me (please note I couldn't review the chinese pages)

**Review by nunix — APPROVED** [review]

File: docs/security/security.md [hunk 2]
```diff
@@ -7,5 +7,5 @@ This section describes the methodology and means of securing a K3s cluster. It's
 
 The documents below apply to CIS Kubernetes Benchmark v1.23.
 
-* [Hardening Guide](hardening-guide.md)
-* [CIS Benchmark Self-Assessment Guide](self-assessment.md)
+* [Hardening Guide](./hardening-guide.md)
+* [CIS Benchmark Self-Assessment Guide](./self-assessment.md)
```
Comment (dereknola) [inline docs/security/security.md:11]: Nit: you don't need this change, the links work just fine.

**Review by dereknola — APPROVED** [review]
I can't give any review of the Translations, overall I am fine with adding support, however, these will not be kept up to date as we lack native speakers for backports.

**Comment by vickyhella** [comment]
> I can't give any review of the Translations, overall I am fine with adding support, however, these will not be kept up to date as we lack native speakers for backports.

I will regularly update the translations and try to keep them up to date.

**Comment by vickyhella** [comment]
@dereknola The PR might need some permissions to run the workflow. Would you please help with it?

File: docs/security/security.md (hunk 2 above)
Comment (vickyhella) [reply docs/security/security.md]: Links updated

**Comment by vickyhella** [comment]
@kingsd041 Would you please help review the Chinese translations? Thanks!

**Review by kingsd041 — APPROVED** [review]
LGTM

**Review by brandond — APPROVED** [review]

**Pull request merged**